.gitignore

```diff
@@ -1,6 +1,7 @@
 build/
 out/
 .vs/
+CmakeSettings.json
```

Comment:
Don't ignore this, we need it to set the default optimization level on Visual Studio